src/Middleware/HeaderPropagation/samples/HeaderPropagationSample/Startup.cs

```diff
@@ -71,19 +74,23 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IHttpCli
                         await context.Response.WriteAsync($"'/' Got Header '{header.Key}': {string.Join(", ", header.Value)}\r\n");
                     }
 
-                    await context.Response.WriteAsync("Sending request to /forwarded\r\n");
+                    var clientNames = new[] { "test", "anotehr" };
```

Comment:
typo `another`

Reply:
Took me a while to find why the sample wasn't working as expected 😅